assertk/src/jvmTest/kotlin/test/assertk/JVMAssertFailureTest.kt

```diff
@@ -0,0 +1,29 @@
+package test.assertk
+
+import assertk.assertFailure
+import assertk.assertions.hasMessage
+import assertk.assertions.isEqualTo
+import assertk.assertions.message
+import com.willowtreeapps.opentest4k.AssertionFailedError
+import kotlinx.coroutines.runBlocking
+import test.assertk.assertions.valueOrFail
+import kotlin.coroutines.resume
+import kotlin.coroutines.suspendCoroutine
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
+import kotlin.test.assertFalse
+import kotlin.test.assertSame
+import kotlin.test.assertTrue
+
+class JVMAssertFailureTest {
+    @Test
+    fun can_test_suspending_functions(): Unit = runBlocking {
```

Comment:
This could be in `commonTest` by using `runTest`. I was going to migrate the `assertThat { }` tests to use it, but then with its deprecation it seemed pointless.

If you want I can migrate it in a follow-up.

Reply:
Ah, I forgot they had changed it from `runBlockingTest` and I assumed it wasn't available for common. Will fix.

Reply:
Done